chore: rename RequestResponse to MsgReceived

oetyng opened the pull request:
- This follows the event naming convention and is directly communicating what happened.

bochaco commented on src/routing/dkg/session.rs:
```diff
@@ -52,32 +53,32 @@ impl Session {
-        } else {
-            Some(self.reset_timer())
-        };
+        let add_reset_timer = responses.is_empty();
```
yes nice catch, I'll fix it now

oetyng committed:
chore: rename RequestResponse to MsgReceived

- This follows the event naming convention and is directly communicating
what happened.
